modules/products/bitbucket/nfs/variables.tf

```diff
@@ -18,23 +18,38 @@ variable "capacity" {
 variable "requests_cpu" {
   description = "The minimum CPU compute to request for the NFS instance"
   type        = string
-  default     = "0.25"
+  default     = "1"
```

Comment:
I think we don't have to change the default resources for NFS module. These resources could be overwritten by config file if is needed such as all other resources.

Reply:
Wasn't 0.25 insufficient even as a default value?

Reply:
I think 0.25 is just too small - even with a single product installed, we have the capacity to increase this to 0.5 or 1. We definitely need to increase the `limits` as the current values will cause NFS server to be restored (and I assume result into flakiness of the product).